source/includes/fact-oplog-background.rst

```diff
@@ -4,14 +4,22 @@ to the data on the destination cluster.  When operations
 that ``mongosync`` has not applied roll off the ``oplog`` 
 on the source cluster, the sync fails and ``mongosync`` exits.
 
-During the initial sync, ``mongosync`` may apply operations at a slower
-rate due to the load imposed by copying documents concurrently.
-After ``mongosync`` completes the initial sync, it applies changes 
-faster and is more likely to maintain a position in the ``oplog``
-that is close to the real-time writes occuring on the source cluster.
+Starting in version 1.5.0, {+c2c-product-name+} enables Oplog Rollover
```

Comment:
I think it would be nice to stay consistent because we reference mongosync is the previous paragraphs

```suggestion
Starting in version 1.5.0, mongosync enables Oplog Rollover
```

Reply:
Done!